### ISSUE TITLE
When I was trying to run things from Pycharm, ImportError: No module named 'darkflow.cython_utils.cy_yolo_findboxes' happened

### ISSUE THREAD

**Issue by liuhantao9** [thtrieu/darkflow#546]
I have tried python3 setup.py build_ext --inplace
tried pip install -e .
tried everything, but it just can be solved.
Any help will be appreciated.
<img width="1435" alt="screen shot 2018-01-26 at 1 55 29 am" src="https://user-images.githubusercontent.com/33992579/35434723-f229c960-023c-11e8-84ec-1bcffd80baf8.png">

**Comment by milan-thom** [comment]
I'm facing the same issue too

**Comment by liuhantao9** [comment]
@milan-thom I solved this problem when I degrade my Python to 3.5

**Comment by lijiaze2018** [comment]
@liuhantao9 can you help me that how to use pycharm to run this project？

**Comment by lijiaze2018** [comment]
@ milan-thom can you help me that how to use pycharm to run this project？

**Comment by MananAmin** [comment]
pls  if any1 find solution then post it....

**Comment by lijiaze2018** [comment]
@MananAmin ，I solve it，You should copy the files in the folder cython_utils(when you pip install Cython,the folder will be created) to the \darkflow-master\darkflow\cython_utils folder.if you do this you will solve it.

**Comment by MananAmin** [comment]
@lijiaze2018 did you mean, I copy the library of cython from sit pkg to that cython_utills folder ???

if yes then it's not working  and it doesn't make any sense i think.
pls.  whats app me : 7359251716

**Comment by lijiaze2018** [comment]
@MananAmin Sorry, my friend, I remembered it wrong, so I opened my computer and confirmed it. The correct way is: copy the contents of this  folder：Python35\Lib\site-packages\darkflow\cython_utils\
 to the \darkflow-master\darkflow\cython_utils folder.This time I am sure it is correct.Give you a screenshot of this folder：

**Comment by MananAmin** [comment]
@lijiaze2018  thanks for replying ...
now I get your point but problem is that I don't have installed the first library named darkflow( so can you please upload that four file )

once again thank you.

**Comment by lijiaze2018** [comment]
@MananAmin Of course, but if you don't install darkflow, you may still have problems.
（sorry，I am trying to upload，but GitHub does not support uploading of this format file..........）You can install it, it's very simple, the author has already written it.

**Comment by KLD** [comment]
@kocica posted this on #914 : 
https://stackoverflow.com/questions/52581584/darkflow-no-module-named-darkflow-cython-utils-cy-yolo-findboxes

Worked for me
